cmd/lotus-shed/miner.go

```diff
@@ -354,3 +361,220 @@ var minerUnpackInfoCmd = &cli.Command{
 		}
 	},
 }
+
+var sendInvalidWindowPoStCmd = &cli.Command{
+	Name:        "send-invalid-windowed-post",
+	Usage:       "Sends an invalid windowed post for a specific deadline",
+	Description: `Note: This is meant for testing purposes and should NOT be used on mainnet or you will be slashed`,
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "really-do-it",
```

Comment:
i don't think shed commands need really-do-its, but also no harm in having it...